### PR TITLE
Parepare cran release

### PR DESCRIPTION
**Pull request by wangben718**

### CODE REVIEW

**Review by LittleBeannie — APPROVED** [review]

File: NEWS.md [hunk 1]
```diff
@@ -1,7 +1,33 @@
 # metalite.ae 0.1.2
 
-- Updated `format_ae_specific` to avoid warning when no comparison variables are
-requested.
+## New features
+- `tlf_ae_summary()` and `tlf_ae_specific` now have a new argument `title` for a user-customized title.
```
Comment (nanxstats) [inline NEWS.md:4]: `tlf_ae_specific()` should also have `()` for consistency

File: NEWS.md [hunk 2]
```diff
@@ -1,7 +1,33 @@
...
+- Add new functions: `rate_compare_sum()`, `extend_ae_specific_subgroup()`, `prepare_ae_specific_subgroup()`, `format_ae_specific_subgroup()`, `tlf_ae_specific_subgroup()`, `extend_ae_summary_eaer()`, 
+`format_ae_exp_adj()` and `tlf_ae_exp_adj()`.
+- Add `data/metalite_ae_adex.Rda`.
```
Comment (nanxstats) [inline NEWS.md:7]: For `.rda`, the convention is to use all lower case `.rda`. If you data file name actually ends with `.Rda`, you need to rename it.

File: NEWS.md [hunk 3]
```diff
@@ -1,7 +1,33 @@
...
+- Add new vignettes: 'Subgroup Analysis for Specific AE' and 'Exposure Adjusted Event Rate'.
```
Comment (nanxstats) [inline NEWS.md:8]: It doesn't help to say the titles here. `vignette("vignette-file-name")` will get linked and clickable.

File: NEWS.md [hunk 4]
```diff
@@ -1,7 +1,33 @@
...
+- Add test cases for new functions.
```
Comment (nanxstats) [inline NEWS.md:9]: This belongs to "don't comment on the obvious" and thus should be removed.

File: NEWS.md [hunk 5]
```diff
@@ -1,7 +1,33 @@
...
+- Add system organ class(SOC) information to `outdata` in `prepare_ae_specific()`
```
Comment (nanxstats) [inline NEWS.md:10]: Should be styled as "System Organ Classes (SOC)" (title case and a whitespace)

File: NEWS.md [hunk 6]
```diff
@@ -1,7 +1,33 @@
...
+
+## Bug fixes
```
Comment (nanxstats) [inline NEWS.md:12]: Need a blank line between headings and content.

File: NEWS.md [hunk 7]
```diff
@@ -1,7 +1,33 @@
...
+- Fix bug to avoid warning messages when no comparison variables are requested in `format_ae_specific()`.
```
Comment (nanxstats) [inline NEWS.md:13]: Please link the PR at the end of the sentences following https://style.tidyverse.org/news.html.

File: NEWS.md [hunk 8]
```diff
@@ -1,7 +1,33 @@
...
+- Fix bug to align the row order of output data with the input one in `avg_event()` and `avg_duration()`.
+- Fix bug to display item if AE term has `NA` in `prepare_ae_specific()`.
+- Fix bug to correctly assign treatment labels when a treatment variable has `NA`.
+
+## Improvements
+- Update Github Actions workflow.
```
Comment (nanxstats) [inline NEWS.md:19]: Github -> GitHub

File: NEWS.md [hunk 9]
```diff
@@ -1,7 +1,33 @@
...
+- Add styler workflow.
+- Add CRAN download badge.
+- Remove dependencies on `dplyr` and `tidyr` from `avg_event()`, `avg_duration()` and `fmt_val()`.
+- Remove `R/n_subject.R` and change to use `metalite::n_subject()` in `prepare_ae_specific()`.
+- Race values for adsl and adae are converted to title case in `meta_ae_exmple()`.
+- Rename `R/rtf_output.R` to `R/utility.R`.
+- Remove `R/to_mock.R` and `R/to_sentence.R`, and `to_mock()` and `to_sentence()` are moved to `R/utility.R`.
+- Remove library calls from tests.
+- Add `R/outdata.R` to control `outdata` object.
+- Add an installation section to `README.Rd`.
```
Comment (nanxstats) [inline NEWS.md:29]: `README.md`

File: NEWS.md [hunk 10]
```diff
@@ -1,7 +1,33 @@
...
+- Improve style and formatting for vignettes and codes.
```
Comment (nanxstats) [inline NEWS.md:30]: codes -> code

**Review by nanxstats — CHANGES_REQUESTED** [review]

**Comment by wangben718** [comment]
Hi @fukuhiro2023 , could you please address these issues?

**Comment by fukuhiro2023** [comment]
Hi @wangben718 and @nanxstats , I have updated the `NEWS.md` based on the comments. Please kindly review it.

**Review by nanxstats — APPROVED** [review]
Great, thanks for the update!

**Pull request merged**